### ISSUE TITLE
Support Synology-NAS-Format

### ISSUE THREAD

**Issue by MrDrache333** [damoclark/node-red-contrib-syslog-input#2]
Ive got some error while trying to use your Node with my Synology NAS.
<img width="1487" alt="Bildschirmfoto 2021-03-01 um 17 36 01" src="https://user-images.githubusercontent.com/22854641/109528281-cef81980-7ab4-11eb-9afb-7218aeffc9b3.png">
My configuration is as follows:
<img width="349" alt="Bildschirmfoto 2021-03-01 um 17 36 18" src="https://user-images.githubusercontent.com/22854641/109528319-d91a1800-7ab4-11eb-9c85-b30e1fe1cdbe.png">

Did I do something wrong? Ive heard, that maybe the Tag is missing from Synology? Would it be possible for you to rename the Tag to None or something that I can use your node?

**Comment by damoclark** [comment]
Yes, the problem is the missing tag field.  I may have time to fix this so that its a little more forgiving.  Let me see what I can do.

Damo.
